tests/func/metrics/test_show.py

```diff
@@ -17,23 +17,25 @@ def test_show_simple(tmp_dir, dvc, run_copy_metrics):
     run_copy_metrics(
         "metrics_t.yaml", "metrics.yaml", metrics=["metrics.yaml"],
     )
-    assert dvc.metrics.show() == {"": {"metrics.yaml": 1.1}}
+    assert dvc.metrics.show() == {"workspace": {"metrics.yaml": 1.1}}
```

Comment:
This is a breaking API change, people might rely on this, so normally we can't just break it like that, unless we have a very good reason.